test/e2e/singlecluster/manager_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	config "sigs.k8s.io/kueue/apis/config/v1beta1"
+	testingjobspod "sigs.k8s.io/kueue/pkg/util/testingjobs/pod"
+	"sigs.k8s.io/kueue/test/util"
+)
+
+const (
+	// serviceAccountName created for the project
+	serviceAccountName = "kueue-controller-manager"
+
+	// metricsServiceName is the name of the metrics service of the project
+	metricsServiceName = "kueue-controller-manager-metrics-service"
+)
+
+var _ = ginkgo.Describe("Metrics", func() {
+	ginkgo.It("should ensure the metrics endpoint is serving metrics", func() {
+		ginkgo.By("Creating the curl-metrics pod to access the metrics endpoint")
+		pod := testingjobspod.MakePod("curl-metrics", config.DefaultNamespace).
+			ServiceAccountName(serviceAccountName).
+			Image(util.E2eTTestCurlImage, []string{
+				"/bin/sh", "-c", fmt.Sprintf(
+					"curl -s -k -H \"Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\" https://%s.%s.svc.cluster.local:8443/metrics",
```

Comment:
I'm wondering about alternatives to curl here, because this approach requires a new image.
Can we use `kubectl get --raw /metrics` instead?